docs/runtime/transfer_learning.md

```diff
@@ -0,0 +1,199 @@
+# Transfer Learning
+
+Let's classify dogs and cats using the pre-trained model. It will be `mobilenet v2` in our case.
+
+## Preparing dataset
+The first step is conversion dataset (input data and expected outputs) to the binary format accepted by `ONE`. Note, that for many cases it can be achieved via [tf_dataset_converter](https://github.com/Samsung/ONE/tree/master/tools/generate_datafile/tf_dataset_converter).
+
+```python
+import os
+import tensorflow as tf
+import pathlib
+import numpy as np
+
+_URL = 'https://storage.googleapis.com/mledu-datasets/cats_and_dogs_filtered.zip'
+path_to_zip = tf.keras.utils.get_file('cats_and_dogs.zip', origin=_URL, extract=True)
+PATH = os.path.join(os.path.dirname(path_to_zip), 'cats_and_dogs_filtered')
+
+train_dir = os.path.join(PATH, 'train')
+validation_dir = os.path.join(PATH, 'validation')
+
+BATCH_SIZE = 32
+IMG_SIZE = (160, 160)
+
+def label_to_array(label):
+    arr = np.zeros(2, dtype=float)
+    arr[label] = 1.
+    tensor = tf.convert_to_tensor(arr, tf.float32)
+    return tensor
+
+train_dataset = tf.keras.preprocessing.image_dataset_from_directory(train_dir,
+                                                            shuffle=True,
+                                                            batch_size=BATCH_SIZE,
+                                                            image_size=IMG_SIZE)
+
+current_dir = pathlib.Path(__file__).parent
+with open(current_dir/'cats_and_dogs.input.bin', 'wb') as in_stream:
+    with open(current_dir/'cats_and_dogs.output.bin', 'wb') as out_stream:
+        for image,label in train_dataset:
+            # print(image.numpy().dtype)
+            in_stream.write(image.numpy().astype(np.float32).tobytes())
+            # print(label)
+            out_stream.write(label.numpy().astype(np.float32).tobytes())
+```
+
+## Feature extraction
+The first step to align the model to our problem can be inserting customs ops on the top of the model. The base of the model (with unchanged weights) remains the same.
+
+```python
+import tensorflow as tf
+import numpy as np
+
+BATCH_SIZE = 32
+IMG_SIZE = (160, 160)
+IMG_SHAPE = IMG_SIZE + (3,)
+
+base_model = tf.keras.applications.MobileNetV2(input_shape=IMG_SHAPE,
+                                               include_top=False,
+                                               weights='imagenet')
+
+base_model.trainable = False
+image_batch = tf.zeros([32, 160, 160, 3])
+feature_batch = base_model(image_batch)
+
+# feature extraction
```

Comment:
How about this comment? 

For me, It was hard to catch the meaning of `feature extraction`.
So I checked the reference link (you linked under) - it said base_model do feature extraction and additional layer do classification. So I suggest under comments.  

>  By specifying the include_top=False argument, you load a network that doesn't include the classification layers at the top, which is ideal for feature extraction. 
( https://www.tensorflow.org/tutorials/images/transfer_learning#:~:text=By%20specifying%20the%20include_top%3DFalse%20argument%2C%20you%20load%20a%20network%20that%20doesn%27t%20include%20the%20classification%20layers%20at%20the%20top%2C%20which%20is%20ideal%20for%20feature%20extraction.) 

```suggestion
# define classification layer 
```

```suggestion
# define additional layer 
```

Reply:
right, I see, how about `define custom classification layers`?

Reply:
Looks great!